ds/data/processed/defra/air_pollution_mean_pm10.lep.yaml

```diff
@@ -0,0 +1,39 @@
+api_doc_url: https://uk-air.defra.gov.uk/data/pcm-data
+api_type: FETCH
+description_short: Mean PM10 particulate background pollution
+description: Mean PM10 particulate background pollution data for LEP regions aggregated from 1km x 1km resolution UK data modelled by DEFRA.
+endpoint_url:
+  - https://uk-air.defra.gov.uk/datastore/pcm/mappm102007g.csv
```

Comment:
In order for the data provenance panel to be easiert to read I've recently changed this to object format with years as keys https://github.com/nestauk/beis-indicators/blob/dev/ds/data/processed/defra/air_pollution_mean_pm10.yaml#L6 so it would be good to use the same format for consistency.

<img width="591" alt="Screenshot 2020-04-29 at 18 12 41" src="https://user-images.githubusercontent.com/1309648/80625496-384a6e80-8a45-11ea-8526-cb73a9d53824.png">